pkg/sfu/sequencer.go

```diff
@@ -44,8 +47,10 @@ type packetMeta struct {
 	// the same packet.
 	// The resolution is 1 ms counting after the sequencer start time.
 	lastNack uint32
+	// number of NACKs this packet has received
+	nacked uint8
```

Comment:
New field to keep track how many NACKs a sequence number has gotten. Actually, this is not truly how many it has received. This is the counter of how many times we are replying to a NACK. As there are interval conditions in this module which decides when to reply to a NACK with a retransmission, this counter is not strictly number of NACKs receiver. Rather it is the number of NACKs acknowledged.